src/videojs-http-streaming.js

```diff
@@ -350,22 +354,37 @@ class HlsHandler extends Component {
       if (!_player.hasOwnProperty('hls')) {
         Object.defineProperty(_player, 'hls', {
           get: () => {
-            videojs.log.warn('player.hls is deprecated. Use player.tech().hls instead.');
+            videojs.log.warn('player.hls is deprecated. Use player.vhs instead.');
+            tech.trigger({ type: 'usage', name: 'vhs-player-access' });
```

Comment:
I think player.tech().vhs is fine (and makes sense), but I'd just be a bit worried that we do have a decent number of APIs and recommended calls that are at the vhs level (rather than at the video.js level) and if we recommend using player.tech().vhs people are going to see lots of warning logs in the console:

```
VIDEOJS: WARN: Using the tech directly can be dangerous. I hope you know what you're doing.
See https://github.com/videojs/video.js/issues/2617 for more info.
```

Which might seem confusing to users (and potentially bothersome). Should we consider removing that log message in video.js?

Reply:
At the very least, we should make a better landing page for that log. That currently points to the v5 related issue that recommends `IWillNotUseThisInPlugins`. We can create a page on the wiki (or on the videojs.com site) that says something along the lines of "be careful of doing stuff on the tech, pass in a value like `true` to suppress the warning".

I do think the warning is useful because a lot of people copy snippets of code without looking into the code. Hopefully, that warning is helping people think more about it.